web/src/components/button.js

```diff
@@ -22,10 +24,29 @@ export default function Button({
         isSecondary ? secondaryStyle : ''
     }${isSecondaryDark ? secondaryDarkStyle : ''}`;
 
+    if (to) {
+        return (
+            <Link to={to} className={finalStyles}>
+                Learn More
+            </Link>
+        );
+    }
+    if (href) {
+        return (
+            <a
+                className={finalStyles}
+                href={href}
+                target="_blank"
+                rel="noopener noreferrer"
+            >
+                Learn More
+            </a>
+        );
+    }
     return (
-        <Link to="/" className={finalStyles}>
+        <button type="button" className={finalStyles}>
             Learn More
-        </Link>
+        </button>
```

Comment:
Why change this to a button?

Reply:
it's only a button if its not a Link element or an a link element

Reply:
What does the button do?

Reply:
If it is used as a button, it will be a button. A button would be used in a form or trigger something on the page. as its being used throughout the app, i included in case it needs to be a button @asasmith